package.json

```diff
@@ -32,6 +38,7 @@
     "nodemailer": "^6.4.8",
     "pdfkit": "^0.11.0",
     "randexp": "^0.5.3",
+    "read-pkg-up": "^7.0.1",
```

Comment:
It was installed for experimental purpose, and currently has been removed from our actual code base.
It will be removed from package.json as well. For tracking this in future, we have created an issue [https://github.com/googleinterns/OAS-Adherence-Test-Suite/issues/5](url).